relay/common/src/lib.rs

```diff
@@ -112,5 +116,25 @@ pub fn relay_era_payout(params: EraPayoutParams) -> (Balance, Balance) {
 	(staking_payout, rest)
 }
 
+pub struct LocationAsSuperuser<SuperuserLocation, RuntimeOrigin>(
+	PhantomData<(SuperuserLocation, RuntimeOrigin)>,
+);
+impl<SuperuserLocation: Get<Location>, RuntimeOrigin: OriginTrait> ConvertOrigin<RuntimeOrigin>
```

Comment:
for matching we use `Contains`

```suggestion
impl<SuperuserLocation: Contains<Location>, RuntimeOrigin: OriginTrait> ConvertOrigin<RuntimeOrigin>
```

Reply:
Fixed in https://github.com/bkontur/runtimes/pull/12/commits/34963d7b00f96479e271497f988868e7595bcdad